kale/interpret/uncertainty_quantiles.py

```diff
@@ -0,0 +1,1933 @@
+                    if idx_l in ind_landmarks_to_show or ind_landmarks_to_show == [-1]:
+                        if cfg["OUTPUT"]["SAVE_FIGURES"]:
+                            save_location = os.path.join(
+                                save_folder, save_file_preamble + "_errorbound_lm_" + str(idx_l) + ".pdf"
+                            )
+
+                        logger.info("individual errorbound acc for L%s", idx_l)
+
+                        generic_box_plot_loop(
+                            cmaps,
+                            lm_data,
+                            uncertainty_error_pairs,
+                            models_to_compare,
+                            x_axis_labels=x_axis_labels,
+                            x_label="Uncertainty Thresholded Bin",
+                            y_label="Error Bound Accuracy (%)",
+                            num_bins=num_bins_display,
+                            save_path=save_location,
+                            y_lim=120,
+                            width=0.2,
+                            y_lim_min=-2,
+                            font_size_1=30,
+                            font_size_2=30,
+                            show_individual_dots=False,
+                            list_comp_bool=False,
+                        )
+
+        # Plot Jaccard Index
+        if display_settings["jaccard"]:
+            logger.info("Plot jaccard for all landmarks.")
+            if cfg["OUTPUT"]["SAVE_FIGURES"]:
+                save_location = os.path.join(save_folder, save_file_preamble + "_jaccard_all_lms.pdf")
+
+            generic_box_plot_loop(
+                cmaps,
+                all_jaccard_data,
+                uncertainty_error_pairs,
+                models_to_compare,
+                x_axis_labels=x_axis_labels,
+                x_label="Uncertainty Thresholded Bin",
+                y_label="Jaccard Index (%)",
+                num_bins=num_bins_display,
+                save_path=save_location,
+                y_lim=70,
+                width=0.2,
+                y_lim_min=-2,
+                font_size_1=30,
+                font_size_2=30,
+                show_individual_dots=False,
+                list_comp_bool=False,
+            )
+
+            # mean recall for each bin
+            if cfg["OUTPUT"]["SAVE_FIGURES"]:
+                save_location = os.path.join(save_folder, save_file_preamble + "_recall_jaccard_all_lms.pdf")
+
+            generic_box_plot_loop(
+                cmaps,
+                all_recall_data,
+                uncertainty_error_pairs,
+                models_to_compare,
+                x_axis_labels=x_axis_labels,
+                x_label="Uncertainty Thresholded Bin",
+                y_label="Ground Truth Bins Recall",
+                num_bins=num_bins_display,
+                turn_to_percent=True,
+                save_path=save_location,
+                y_lim=120,
```

Comment:
hard-code necessary?

Reply:
No, it isn't. i can change this

Reply:
in discussion: https://github.com/pykale/pykale/discussions/392

Reply:
Stated explicitly?